SCClassLibrary/Common/Streams/FilterPatterns.sc

```diff
@@ -589,6 +589,14 @@ Pbindf : FilterPattern {
 
 
 Pstutter : FilterPattern {
+	var <>n;
+	*new { arg n, pattern;
+		"The use of Pstutter is not recommended. Please use Pdup instead.".warn;
```

Comment:
This will create hundreds of warnings in all kind of projects and force their authors and/or users to do this replacement. In this case, I would prefer if we would allow the language use to change actively because of the correct considerations rather than through pressure.

Reply:
I would also support not generating warnings in this PR, and instead count on the change happen in a more frictionless manner over a possibly longer period of time.

Reply:
Thanks for the comments, apologies for the delay in responding, I've been thinking about this one a lot.

Just to clarify, is this comment regarding all of the warnings, just `Pstutter`, or the types that now warn?

I agree that this will be noisy, but I think it's important that we have a warning of some sort. 

Firstly, users will never receive the old types, which on its own deserves a warning:
```supercollider
Pstutter()
// -> a Pdup
```

And secondly, I'm worried that existing users have little reason to look at the docs for this sort of thing, so only new users will find and use the alternatives, but maybe that is part of a larger discussion on how we communicate this sort of change.

With that in mind, what do you think of the following?
```supercollider
Pstutter : FilterPattern {
   classvar constructNew;

    *initClass {
        constructNew = { |n,pattern|
            "The use of Pstutter is not recommended. Please use Pdup instead.".warn;
            constructNew = {|n,pattern| Pdup(n,pattern)};
            constructNew.(n,pattern);
        };
    }

    *new { |n, pattern|
        ^constructNew.(n,pattern);
    }
}
```
This might look a little odd at first, but it's a strategy pattern, swapping out the behaviour at runtime on first call. 

It prints the warning once and then because there's no conditional, we don't pay for it again. it's as close to zero-cost as I could think of? Also, I'm not keen on `constructNew` as the function name, but it's the best I could come up with so far, any suggestions appreciated!

Many thanks!

Reply:
perhaps 'suggestNew' is a classier name that helps clarify the intent.